docs/technical_tutorial/apollo_best_coding_practice.md

```diff
@@ -1,22 +1,22 @@
-# Best Coding Practice
+# Apollo Best Coding Practice
 
-1. Always compile all, test all and lint all.
+1. Always build, test, and lint all.
 
    ```bash
-   apollo.sh check
+   ./apollo.sh check
    ```
 
-1. Always write unit test and put it along with the source code.
+1. Always write unit tests and put them along with the source files.
 
    ```text
    foobar.h
    foobar.cc
    foobar_test.cc
    ```
 
-1. A bazel target contains at most one header and one source file.
+1. A bazel target should contain at most one header and one source file.
 
-   ```text
+   ```
```

Comment:
Can be marked as python to improve the display.

Reply:
Done.